stories/Icons.story.tsx

```diff
@@ -0,0 +1,341 @@
+import * as React from 'react';
+
+import {
+    description,
+    importExample,
+    header,
+    divider,
+    tab,
+    tabs,
+    title,
+    code as baseCode
+} from 'wix-storybook-utils/Sections';
+
+
+import { allComponents } from './utils/allComponents';
+
+const code = config =>
+    baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+    category: 'Icons',
+    storyName: 'Icons',
+    sections: [
+        header(),
+        tabs([
+            tab({
+                title: 'Icon Inventory',
+                sections: [
+                    description({
+                        title: 'Importing the icons',
+                        text: [
+                            'We are using the ',
+                            <a href="https://www.npmjs.com/package/@svgr/webpack" >
+                                svgr plugin
+                            </a>,
```

Comment:
That's not true
Not all our consumers use Yoshi, and not all versions of yoshi support svgr

Reply:
and also, older versions of yoshi didn't support using svgr on the server side, which caused their apps to break.
Generally this shouldn't be a problem for OOI TPA's, but it might be for iframe TPA's